rosdep/base.yaml

```diff
@@ -3401,6 +3402,7 @@ libpqxx-dev:
   ubuntu:
     artful: [libpqxx-dev]
     bionic: [libpqxx-dev]
+    focal: [libpqxx-dev]
```

Comment:
It looks like this has been `libpqxx-dev` all the way back to trusty.  With that in mind, I'll suggest that for future compatibility we change this entire block to:

```
ubuntu: [libpqxx-dev]
```